helm/values-uat.yaml

```diff
@@ -39,14 +39,14 @@ microservice-chart:
     httpGet:
       path: /actuator/health/liveness
       port: 8080
-    initialDelaySeconds: 60
+    initialDelaySeconds: 80
```

Comment:
Do you need to make this change? 👀
Was made for preventive purposes?

Reply:
thank you! I have to configure to 60 seconds